src/component-index.js

```diff
@@ -26,10 +26,6 @@ limitations under the License.
 
 module.exports.components = {};
 module.exports.components['structures.CreateRoom'] = require('./components/structures/CreateRoom');
-module.exports.components['structures.login.ForgotPassword'] = require('./components/structures/login/ForgotPassword');
```

Comment:
aside: it would be nice to fix `reskindex` so that it doesn't rewrite half the index depending on which OS you happen to be using...